packages/web-components/src/AgoricWalletConnection.js

```diff
@@ -133,7 +133,7 @@ export const makeAgoricWalletConnection = (makeCapTP = defaultMakeCapTP) =>
           return this._bridgePK.promise;
         },
         reset: () => {
-          this.reset();
+          void this.reset();
```

Comment:
Is this explicitly ignoring a dangling promise?  This idiom seems like a good way to indicate that situation.